### PR TITLE
feat(hub-common): add token to schedule api endpoints

### PR DESCRIPTION
**Pull request by sansth1010**
1. Description: This PR adds  authenticated user `token` to content update schedule API endpoint. Soon, the schedule API will only accept request that have valid token in query parameter. PR implementing token in API endpoint is [here](https://github.com/ArcGIS/hub-download-api/pull/36). This PR has to be merged before the backend API PR. 

1. Instructions for testing:

1. Closes Issues: #[9348](https://devtopia.esri.com/dc/hub/issues/9348)

1. [x] Updated meaningful TSDoc to methods including Parameters and Returns, see [Documentation Guide](https://esri.github.io/hub-components/storybook/?path=/story/guides-documentation--page)

1. [x] used semantic commit messages
  
1. [x] PR title follows semantic commit format (**CRITICAL** if the title is not in a semantic format, the release automation will not run!)

1. [ ] updated `peerDependencies` as needed. **CRITICAL** our automated release system can **not** be counted on to update `peerDependencies` so we _must_ do it manually in our PRs when needed. See the [updating peerDependencies](/RELEASE.md#Updating-peerDependencies) section of the release instructions for more details.

### CODE REVIEW

**Review by abp6318 — APPROVED** [review]
Tested locally alongside the force update button to confirm nothing is broken and I think we're good! I would wait for a review from @sonofflynn89 just to be safe, but this looks good to me!

😄

**Review by sonofflynn89 — APPROVED** [review]

**Pull request merged**
